src/controllers/login.ts

```diff
@@ -0,0 +1,53 @@
+import express, { Router,NextFunction, Request, Response } from 'express';
+import jwt from 'jsonwebtoken';
+
+//Cambiar a clases
+
+export class LoginController {
+
+    res:Response;
+    req:Request;
+
+    constructor (res:Response, req: Request) {
+        this.res = res;
+        this.req = req;
+    }
+
+    loginUser():boolean {
+        // JWT with regular user role
+         const token = jwt.sign( {
```

Comment:
Use `prettier` to format the code properly.

Reply:
done